routers/repo/blame.go

```diff
@@ -236,11 +232,11 @@ func renderBlame(ctx *context.Context, blameParts []git.BlamePart, commitNames m
 				lineNumbers.WriteString(fmt.Sprintf(`<span id="L%d" data-line-number="%d"></span>`, i, i))
 			}
 
-			//Code line
-			line = gotemplate.HTMLEscapeString(line)
 			if i != len(lines)-1 {
 				line += "\n"
 			}
+			fileName := fmt.Sprintf("%v", ctx.Data["FileName"])
```

Comment:
Why %v is needed here?

Reply:
Did this to avoid
```
 cannot use ctx.Context.Data["FileName"] (type interface {}) as type string in argument to highlight.File: need type assertion
```
There is probably a better way I just didn't know and used this